domains/hjgaming.json

```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "Hjgaming"
+  },
+  "record": {
+    "TXT": [
+      "vc-domain-verify=hjgaming.part-of.my.id,bc8b061634e2b7099c52"
```

Comment:
For the time being we do not support Vercel as we need to be in the [PSL](https://en.wikipedia.org/wiki/Public_Suffix_List). We will apply only if we get demand, so in the mean time please use another service.